Fortran90/data_types/main.f90

```diff
@@ -31,7 +31,7 @@ program main
         ! Calculating the self-consistent correction fields
         check=.false.
         call solver(obj_approx%hamilt%corr%value,npar,check)
-        
+    
         ! Coverage vs Chemical Potential Plot
         chemp=-1.40d0
         do i=1,240
```

Comment:
Let the user specify initial and final chemical potential, and the number of points (or just the Dmu). Then have the code calculate number of points.